docs/test-pages/TCD-003.adoc

```diff
@@ -0,0 +1,23 @@
+= TCD-003
+
+== Purpose
+
+Verifying the statement definition rules in the IDE (Eclipse or idea).
+
+=== Tested requirements
+
+REQ-SYNT-002
+
+=== Used JSL model
+
+xref:resources/shop2.jsl[shop2.jsl]
+
+== Test scenario
+
+. Open the xref:resources/shop2.jsl[shop2.jsl] with the IDE.
+
+. The IDE is handling and showing the opened file as a JSL file (Syntax highlight is OK.), and there are no errors.
```

Comment:
Again, we can test IDE plugins, but not IDEs. We should phrase every test in a way where this is absolutely clear to everyone.